background/sync-manager.js

```diff
@@ -85,19 +86,32 @@ const syncMan = (() => {
       return ctrl.put(...args);
     },
 
+    async setDriveOptions(driveName, options) {
+      const key = `secure/sync/driveOptions/${driveName}`;
+      await browser.storage.sync.set({
```

Comment:
chromeSync.setValue(key, options)

and add chromeSync to `global` comment after chromeLocal.